base/error.jl

```diff
@@ -227,11 +247,16 @@ ERROR: AssertionError: 3 is an odd number!
 julia> @assert isodd(3) "What even are numbers?"
 ```
 """
-macro assert(ex, msgs...)
-    msg = isempty(msgs) ? ex : msgs[1]
+macro assert(ex, args...)
+
+    aargs = filter(e -> !isexpr(e, :(=)), args)
+    aakws = filter(e -> isexpr(e, :(=)), args)
```

Comment:
Minor point, but what do these names mean? I'm guessing "aargs" = "assert args" but "aakws" has me stumped.

Reply:
that was the names used in an old stack overflow post about kwargs in macros :)

I updated the PR to have a more descriptive name for the variables.